esp32/modules/boot.py

```diff
@@ -38,15 +38,18 @@
         ugfx.clear(ugfx.WHITE)
         ugfx.flush(ugfx.LUT_FULL)
 except BaseException as e:
+    sys.print_exception(e)
+    import easydraw
+    easydraw.msg("A fatal error occured!","Still Crashing Anyway", True)
+    easydraw.msg("")
+
     # if we started the splash screen and it is not the default splash screen,
     # then revert to original splash screen.
     if splash == badge.nvs_get_str('boot', 'splash', 'splash') and splash != 'splash':
+        easydraw.msg("Disabling custom splash screen.")
```

Comment:
My original patch had 'disabled' instead of 'disabling', but since this is an action and we're actually changing the config, I think 'disabling' makes more sense.